docs/architecture/c-sharp-client-library.md

```diff
@@ -71,13 +71,73 @@ There will be two handlers for each of the data types returned:
 - `EmissionsHandler`
 - `ForecastHandler`
 
-Each is responsible for interacting on its own domain. For instance, EmissionsHandler can have a method `GetAverageCarbonIntensityAsync()` to pull EmissionsData data from a configured data source and calculate the average carbon intensity. ForecastHandler can have a method `GetCurrentAsync()`, that will return a EmissionsForecast instance.
+Each is responsible for interacting on its own domain. For instance, EmissionsHandler can have a method `GetAverageCarbonIntensityAsync()` to pull EmissionsData data from a configured data source and calculate the average carbon intensity. ForecastHandler can have a method `GetCurrentForecastAsync()`, that will return a EmissionsForecast instance.
 (**Note**: The current core implementation is using async/await paradigm, which would be the default for library too).
 
 ### Parameters
 
 Both handlers require that exact fields be passed in as input. Within the docs of each library function, we will specifically call out which fields the function expects to be defined versus which are optional. Internally, we will handle creating the CarbonAwareParameters object and validating the fields through that.
```

Comment:
Instead of saying "we will" here (and throughout) just say "we" indicating that it is already handled, not soon to be handled